core/chains/evm/config/toml/defaults/Metis_Rinkeby.toml

```diff
@@ -11,7 +11,7 @@ Enabled = true
 [GasEstimator]
 Mode = 'SuggestedPrice'
 PriceMin = '0'
-BumpThreshold = 0
+BumpThreshold = 3
```

Comment:
The fallback `BumpThresholed` is already 3, so these overrides can be omitted.
https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/config/toml/defaults/fallback.toml#L40